core/src/main/scala/pink/cozydev/lucille/QueryPrinter.scala

```diff
@@ -21,7 +21,7 @@ import cats.data.NonEmptyList
 
 object QueryPrinter {
 
-  def print(query: Query): String = {
+  def print(query: Query, precision: Int = 2): String = {
```

Comment:
discussed on discord with Andrew and I think it's worth a scaladoc just to explain that this precision isn't related to the original precision in the boost query. It is user's choice so it's not a problem but it's something a user should be aware of explicitly